allennlp/semparse/worlds/nlvr_box.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
```

Comment:
Are these classes _only_ used in the NLVR executor?  If they are, I'd probably just put them in the executor module.

Reply:
They are also used in the `NlvrWorld`, and they used to be in that module earlier. The reason I moved them out is because there is now a circular dependency (`NlvrWorld` depends on `NlvrExecutor`, and `NlvrExecutor` depends on `Box` and `Object`).